src/einsteinpy/symbolic/einstein.py

```diff
@@ -90,3 +95,36 @@ def change_config(self, newconfig="ul", metric=None):
             new_tensor, self.syms, config=newconfig, parent_metric=metric
         )
         return new_obj
+
+    def lorentz_transform(self, transformation_matrix):
```

Comment:
all the code is already there in its parent class. I was expecting something more like this.
https://github.com/einsteinpy/einsteinpy/blob/master/src/einsteinpy/symbolic/vector.py#L95-L98